pkg/specs/podspec_diff.go

```diff
@@ -116,10 +116,17 @@ func compareMaps[V comparable](current, target map[string]V) (bool, string) {
 	return true, ""
 }
 
+func isCurrentVolumeToBeIgnored(name string) bool {
```

Comment:
This is very nice English, perhaps a bit too poetic for code :)

```suggestion
func shouldVolumeBeIgnored(name string) bool {
```